### DISCUSSION TITLE
단축 평가란 무엇인가요?

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#33] [타입변환과 단축 평가]
- 단축 평가란 무엇인가요?
  - 단축 평가를 사용해본 경험이 있나요?

**Comment by pipisebastian** [comment]
>단축 평가란 무엇인가요 

표현식을 평가하는 도중에, 평가 결과가 확정된 경우, 나머지 평가과정은 생략하는 것입니다.

>단축 평가를 사용해본 경험이 있나요?

논리합(||), 논리곱(&&) 연산자를 사용했는데, 주로 조건문에서 모든 조건이 다 만족하는지, 혹은 하나라도 만족하는지를 검사할때 사용했습니다

**Comment by publdaze** [comment]
> - 단축 평가란 무엇인가요?
- 표현식 평가할 때 평가 결과 확정되면 나머지 평가 생략하는 것
> - 단축 평가를 사용해본 경험이 있나요?
- 리액트 프로젝트에서 api로 받아온 값이 없을 경우 컴포넌트를 렌더링 하지 않을 때 사용해봄
